File: FSharp/HttpClient/WebApplication/Startup.fs
namespace WebApplication

open System
open Microsoft.AspNetCore.Builder
open Microsoft.AspNetCore.Hosting
open Microsoft.AspNetCore.Http
open Microsoft.Extensions.DependencyInjection
open Microsoft.Extensions.Hosting

type Startup() =

    member this.ConfigureServices(services: IServiceCollection) =
        ()

    member this.Configure(app: IApplicationBuilder, env: IWebHostEnvironment) =
        if env.IsDevelopment() then
            app.UseDeveloperExceptionPage() |> ignore

        app.UseRouting() |> ignore

        app.UseEndpoints(fun endpoints ->
            endpoints.MapGet("/", fun context -> context.Response.WriteAsync("Get!")) |> ignore
            endpoints.MapPost("/", fun context -> context.Response.WriteAsync("Post!")) |> ignore
            endpoints.MapPut("/", fun context -> context.Response.WriteAsync("Put!")) |> ignore
            endpoints.MapDelete("/", fun context -> context.Response.WriteAsync("Delete!")) |> ignore
            ) |> ignore

File: CSharp/SignalRTesting/Server/Startup.cs
using System.Threading.Tasks;
using Microsoft.AspNetCore.Builder;
using Microsoft.AspNetCore.Hosting;
using Microsoft.Extensions.DependencyInjection;
using Microsoft.Extensions.Hosting;

namespace Server
{
    public class Startup
    {
        public void ConfigureServices(IServiceCollection services)
        {
            // services.AddCors(cors => cors.AddDefaultPolicy(policy =>
            //     policy
            //         .AllowAnyHeader()
            //         .AllowAnyMethod()
            //         .AllowCredentials()
            //         .SetIsOriginAllowed(_ => true)
            // ));

            services.AddSignalR()
                .AddMessagePackProtocol();
        }

        public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
        {
            if (env.IsDevelopment()) app.UseDeveloperExceptionPage();

            // app.UseCors();

            app.UseStaticFiles();

            app.UseRouting();
            app.UseEndpoints(endpoints =>
            {
                endpoints.MapGet("/", httpContext =>
                {
                    httpContext.Response.Redirect("/client.html");
                    return Task.CompletedTask;
                });
                endpoints.MapHub<TestHub>("/test");
            });
        }
    }
}
